lambdas/stac-gen/handler.py

```diff
@@ -14,11 +14,9 @@
     "s3",
 )
 
-
 STAC_DB_HOST = os.environ.get("STAC_DB_HOST")
 STAC_DB_USER = os.environ.get("STAC_DB_USER")
-STAC_DB_PASSWORD = os.environ.get("STAC_DB_PASSWORD")
-
+PGPASSWORD = os.environ.get("PGPASSWORD")
```

Comment:
Nit: in create_item on line 39, can we add a method to strip the file extension out of the stac id?
Something like 
```
item_id = os.path.basename(cog_url).strip(".hd5.tiff.orwhatever")
```